oms/src/main/java/com/gfg/oms/client/Client.java

```diff
@@ -0,0 +1,116 @@
+package com.gfg.oms.client;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.exc.MismatchedInputException;
+import com.gfg.oms.Config;
+import com.gfg.oms.Request;
+import com.gfg.oms.Response;
+import com.gfg.oms.consignment.request.Create;
+import com.gfg.oms.response.Error;
+import com.gfg.oms.consignment.response.CreateConsignmentResponse;
+import lombok.AllArgsConstructor;
+
+import javax.validation.constraints.NotNull;
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+
+@AllArgsConstructor
+public class Client {
```

Comment:
Logic of the client is:
- `getResponseContent` and `getConnection` are low-level methods, throws exception
- `sendRequest` always return DTO with data about what was the result of http communication. Handles network issues.
- `doTypeOfOperationRequest` gets DTO (RawResponse) and process it specifically for each method: map to certain classes, check validity.